repository/c.json

```diff
@@ -1065,6 +1065,18 @@
 				}
 			]
 		},
+		{
+			"name": "Chrome Extension Exporter",
+			"description" : "Exports Chrome Extension files from the current project as ZIP for quick local distribution.",
```

Comment:
It's easier to manage if you leave this out. Package control will then take the GitHub repo description (which you can easily change). However, if you want this to be different, that's also fine of course.